src/SmartComponents/BaselinesPage/DeleteBaselinesModal/DeleteBaselinesModal.js

```diff
@@ -44,7 +44,9 @@ export class DeleteBaselinesModal extends Component {
 
     render() {
         const { baselineId, modalOpened, selectedBaselineIds, toggleModal } = this.props;
-        const deleteMessage = baselineId || selectedBaselineIds.length === 1 ? 'Delete baseline' : 'Delete baselines';
+        const deleteMessage = baselineId || selectedBaselineIds.length === 1
+            ? `Delete ${selectedBaselineIds.length} selected baseline`
```

Comment:
`${selectedBaselineIds.length}` here could be replaced by "1" for clarity, but it's very minor. I'll leave it up to you if it's worth having to regenerate snapshots :grin:

Reply:
Sure it is 😆